### PR TITLE
UI: Add source table and status for generated dataproc jobs

### PR DESCRIPTION
**Pull request by forksumit**
Add source table and dataproc job status along with dataproc job hyperlinks in Dataproc Jobs section in prepare-migration page

### CODE REVIEW

File: conversion/conversion.go [hunk 1]
```diff
@@ -292,10 +296,32 @@ func processDataWithDataproc(sourceProfile profiles.SourceProfile, targetProfile
...
 		}
 
-		id, err := dproc.TriggerDataprocTemplate(batchClient, srcTable, srcSchema.Schema, strings.Join(primaryKeys, ","), dataprocRequestParams)
+		op, err := dproc.TriggerDataprocTemplate(ctx, batchClient, srcTable, srcSchema.Schema, strings.Join(primaryKeys, ","), dataprocRequestParams)
+		if err != nil {
+			logger.Log.Error(fmt.Sprintf("Failing to trigger Dataproc template for %s.%s", srcSchema.Schema, srcTable))
+			return err
+		}
+		meta, _ := op.Metadata()
+		batchName := meta.Batch
```
Comment (bharadwaj-aditya) [inline conversion/conversion.go:305]: nit: if the variables are not used anywhere else, merge these into 1 line

**Review by bharadwaj-aditya — APPROVED** [review]
LGTM

**Review by shreyakhajanchi — APPROVED** [review]
LGTM

File: conversion/conversion.go (hunk 1 above)
Comment (forksumit) [reply conversion/conversion.go]: Since `op.Metadata()` returns two values, had to assign it to a variable first.

**Pull request merged**